presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -834,6 +834,18 @@ Log the stats equivalent plan and canonicalized plans used in history based opti
 Enable analysis and propagation of logical properties like distinct keys or cardinality among the nodes of
 a query plan. The optimizer may then use these properties to perform various optimizations.
 
+``optimizer.use-histograms``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``boolean``
```

Comment:
Please unindent this section to remove the vertical line in the left of this screenshot. 
<img width="823" alt="Screenshot 2024-06-20 at 3 03 23 PM" src="https://github.com/prestodb/presto/assets/7013443/663cefb6-021c-4596-8f2f-ba0b4a615faa">